### ISSUE TITLE
Additional XMP support

### ISSUE THREAD

**Issue by donmccurdy** [donmccurdy/glTF-Transform#476]
Support for `KHR_xmp_json_ld` was added in #468. This issue tracks some additional functionality that would be nice to have in the XMP support, but which is not part of the feature at launch:

- [ ] Pre-populate existing data for interactive mode
- [ ] Provide CLI option (`--no-xmp-update`) to skip updating `xmp:ModifyDate` automatically
- [ ] Generate per-scene (or pantry/ingredient?) XMP packets in CLI 'merge' command
- [ ] Further unit test coverage of CLI 'xmp' command<sup>1</sup> 

<sup>1</sup> Current tests cover the scripting interface only. The CLI relies heavily on interactive prompts, so that is harder to cover with automated tests.

**Comment by donmccurdy** [comment]
/cc @lexaknyazev @weegeekps FYI, I'm tracking the suggestion of adding something like a `--no-xmp-update` flag, and some other ideas, here. Currently the CLI does not add `KHR_xmp_json_ld` to an asset that doesn't already use it, so there are relatively few assets in the wild likely to be affected. I'm inclined to wait on the tasks above until getting more feedback from users, and that won't happen until the new feature is announced to users more broadly I imagine. I'd like to put together a post (for Twitter at least) fairly soon. Other suggestions welcome, either here or new issues, in the meantime.

**Comment by donmccurdy** [comment]
Closing the ticket for now — I'm open to adding any of these features, but haven't heard from others that they're needed so far.
